ambry-commons/src/main/java/com/github/ambry/mysql/MySqlDataAccessor.java

```diff
@@ -317,7 +286,7 @@ private boolean isBetterEndpoint(DbEndpoint first, DbEndpoint second) {
    * Close a resource without throwing exception.
    * @param resource the resource to close.
    */
-  private static void closeQuietly(AutoCloseable resource) {
+  private void closeQuietly(AutoCloseable resource) {
```

Comment:
Why not static?  (This can probably move to MySqlUtils anyway.)

Reply:
I can revert the change.